inputremapper/injection/mapping_handlers/combination_handler.py

```diff
@@ -133,18 +139,55 @@ def notify(
                 suppress = False
             event = event.modify(value=0)
 
-        if suppress:
-            return False
+        if not suppress:
+            if is_activated:
+                # Send key up events to the forwarded uinput if configured to do so.
+                self.forward_release()
+
+            logger.debug("Sending %s to sub-handler", self.mapping.input_combination)
+            self._output_active = bool(event.value)
+            sub_handler_result = self._sub_handler.notify(event, source, suppress)
+
+            if is_pressed:
+                # Only if the sub-handler return False, we need a release-event later.
+                # If it handled the event, the user never sees this key-down event.
+                self.require_release_later(not sub_handler_result, event)
+                return sub_handler_result
```

Comment:
`is_activated` and `is_pressed` must be equal at this point.
`is_activated==False` and `is_pressd==True` can only happen on transition 1 which is already covered on line 116

Reply:
and `is_activated==True` and `is_pressed==False` is just impossible